API/harbor/swagger.yaml

```diff
@@ -4871,7 +4871,14 @@ definitions:
         description: Correspond to the UI about whether the project's publicity is  updatable (for UI)
       current_user_role_id:
         type: integer
-        description: The role ID of the current user who triggered the API (for UI)
+        description: The role ID with highest permission of the current user who triggered the API (for UI)
+        deprecated: true
+      current_user_role_id_list:
```

Comment:
I thinks `current_user_role_ids` better than `current_user_role_id_list`. Just like compare with `projects` and `project_list`, `projects` will be better.

Reply:
Sure, I was just thinking `current_user_role_ids` looks too similar to `current_user_role_id`